docs/Getting-Started-TPU-VM.md

```diff
@@ -124,18 +131,11 @@ python infra/launch.py -- python levanter/src/levanter/main/train_lm.py --config
 ### Babysitting Script
 
 If you are using a preemptible TPU VM, you probably want to use the "babysitting" script that automatically re-creates
-the VM. This is because preemptible instances can be preempted and will always be killed every 24 hours. The babysitting
-script handles both the creation of the node and the running of a job, and also relaunches the TPU VM if it gets preempted.
-It keeps running the command (and relaunching) until the command exits successfully.
-
-Note that the babysitting-script will automatically set the `RUN_ID` environment variable if not set, and pass it to the
-training command. This ensures that restarted jobs have the same run id, which is important for resumes to work.
-
-You can run it like this:
+the VM. This is because preemptible instances can be preempted and will always be killed every 24 hours. You can run `launch.py` with the `--retries` and `--foreground` parameter to accomplish this. If `--retries` is greater than 1, `launch.py` will automatically attempt to re-create the VM and re-run the command if it fails. (`--foreground` is necessary to keep the script from returning immediately.)
```

Comment:
FWIW, a desideratum I'll have after this is for us to use [queued-resources](https://cloud.google.com/tpu/docs/queued-resources) which is a kinda annoying API in the current regime but might be better with this fix in. In this world, you want to pass our setup script as a vm setup script rather than scping it. See #597 / #328 

The advantages of queued-resources are:

* there are now "best effort" vms which are exactly like preemptible AFAICT except they don't get auto-killed every 24 hours.
*  multislice works

The downside is that you maybe have to manually poll if you want to launch a command separately, and you have to poll separately for the VM getting preempted anyway...